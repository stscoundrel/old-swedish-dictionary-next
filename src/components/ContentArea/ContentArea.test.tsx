import ReactDOM from 'react-dom/client'
import renderer from 'react-test-renderer'
import ContentArea from './index'

describe('ContentArea component', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    const root = ReactDOM.createRoot(div)
    root.render(<ContentArea><p>Content</p></ContentArea>)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<ContentArea><p>Content</p></ContentArea>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
