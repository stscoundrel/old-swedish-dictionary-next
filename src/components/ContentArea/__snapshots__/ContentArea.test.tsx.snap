// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentArea component Matches snapshot 1`] = `
<section
  className="section"
>
  <p>
    Content
  </p>
</section>
`;
